Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,46 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    require "pty"
+
+    assert_equal "LGOGDownloader #{version}",
+      pipe_output("#{bin}/lgogdownloader --version").chomp
+
+    login_cmdline =
+      [bin/"lgogdownloader", "--list", "--retries", "1"]
```

Comment:
and then inline them below instead of using splat

Reply:
You mean I should inline the array in both places where it is needed?

Reply:
It's only needed in one place if you dump the `ohai`

Reply:
I feel there is a quite strong case for the `ohai` here, see my explanation. I’ll gladly lose it though if we find another way that doesn’t compromise the UX of `brew test`.

Reply:
Thinking about it, maybe the first test isn’t necessary after all?
If I delete that, the `ohai` can go, too!

Reply:
The version test?

Reply:
Yes, I mean the `--version` test. It’s just a nice to have after all; the functional test covers more.